scripts/postInstall.js

```diff
@@ -108,14 +108,34 @@ module.exports = (context) => {
         const parseString = xml2js.parseString;
         const manifestPath = androidManifestPath + '/AndroidManifest.xml';
         const androidManifest = fs.readFileSync(manifestPath).toString();
-
+        const configFilePath = context.opts.projectRoot + '/config.xml';
         if (androidManifest) {
             parseString(androidManifest, (err, manifest) => {
 
                 if (err) return console.error(err);
 
                 // Getting the package name from the manifest
-                const packageName = manifest['manifest']['$']['package']
+                var packageName = manifest['manifest']['$']['package']
+                if (!packageName) {
+                  try {
+                    let configXmlData = fs.readFileSync(configFilePath).toString();
+                    xml2js.parseString(configXmlData, (error, configData) => {
```

Comment:
result of parseString comes in the callback. The rest of the code should be moved to callback as done [here](https://github.com/juspay/hyper-sdk-cordova/pull/4/files#diff-b9601a097b4f54f7224611d88e3f57e6c9557b382f55db8de095fa359eb91f2fR113)